src/transformers/models/videomae/feature_extraction_videomae.py

```diff
@@ -159,8 +162,20 @@ def __call__(
             videos = [self.resize_video(video, size=self.size, resample=self.resample) for video in videos]
         if self.do_center_crop and self.size is not None:
             videos = [self.crop_video(video, size=self.size) for video in videos]
+
+        # if do_normalize=False, the casting to a numpy array won't happen, so we need to do it here
```

Comment:
Sorry, I don't know this part of the code very well, but why don't we put the following 4 lines in an `else` branch of `if self.do_normalize:`?